src/inject/planner_helper.js

```diff
@@ -86,6 +86,8 @@ PlannerHelper.prototype.makeViewDataButton = function (teacher, course) {
     button.data('teacher', teacher);
     button.on('click', function () {
         plannerHelper.reloadData($(this).data('teacher'), $(this).data('course'));
+        var elem = document.getElementById('planner-helper');
```

Comment:
For consistency with the rest of the code could you use a jQuery selector instead?

    var elem = $('#planner-helper')